lighthouse-core/audits/bootup-time.js

```diff
@@ -72,19 +72,55 @@ class BootupTime extends Audit {
     return result;
   }
 
+  /**
+   * Summarize JS costs by URL
+   * @param {Map<string, Object<string, number>>} executionTimings
+   * @param {LH.Audit.Context} context
+   * @return {{results: Object, totalBootupTime: number, extendedInfo: Object}}
+   */
+  static execCostsByURL(executionTimings, context) {
```

Comment:
should be `executionCostsByURL`, but from the outside at least that doesn't appear to be any different than `getExecutionTimingsByURL`?